xla/service/gpu/nccl_api.h

```diff
@@ -250,8 +270,12 @@ class NcclApi {
   // https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/api/comms.html#ncclcommderegister
   virtual absl::StatusOr<NcclRegisteredBufferHandle> DeregisterBuffer(
       NcclCommHandle comm, NcclRegisteredBufferHandle handle) = 0;
-};
 
+  // Returns if initializing communicator with custom configuration is supported
+  // or not. Available after 2.17.0.
+  // https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/api/comms.html#ncclcomminitrankconfig
+  virtual absl::StatusOr<bool> IsInitConfigSupported() = 0;
```

Comment:
Do we ever compile with NCCL that old? Definitely not at google. Should we just assume config is available?

Reply:
I'm not sure about other users though but we always compile against the latest. If there's no concern on your side, i think we can make that assumption.